src/ray/core_worker/core_worker.cc

```diff
@@ -415,6 +421,13 @@ CoreWorker::CoreWorker(const CoreWorkerOptions &options, const WorkerID &worker_
                           : std::shared_ptr<LeasePolicyInterface>(
                                 std::make_shared<LocalLeasePolicy>(rpc_address_));
 
+  std::function<uint64_t()> get_max_pending_lease_requests_per_scheduling_category =
+      [this]() -> uint64_t {
+    if (RayConfig::instance().max_pending_lease_requests_per_scheduling_category() > 0) {
+      return RayConfig::instance().max_pending_lease_requests_per_scheduling_category();
+    }
+    return std::max<int64_t>(kMinConcurrentLeaseCap, num_alive_nodes_.load());
```

Comment:
why is this typed as `int64_t`? also why didn't any linter catch this implicit lossy type conversion? seems bad